deployments/managed-services.mdx

```diff
@@ -0,0 +1,21 @@
+---
+title: "Managed services"
+---
+
+One of the key features of Architect.io is its dependency graph, which is a visual representation of the relationships between different services in an application.
+The dependency graph is a useful tool for managing complex applications with multiple services and dependencies, and demonstrates how those services work together in the application.
+
+Let's consider an example of a simple web application with a single `hello-world` service.
+The dependency graph for this application would have two nodes: one for the `hello-world` service and one for the Traefik `gateway`.
+
+![](/images/dependency-graph.png)
+
+# Gateway Node (traefik)
```

Comment:
This header and "Cert manager" below should be lower-level headers. They should probably use `##`